tests/integration/cocoon/test_cocoon_transactions.py

```diff
@@ -600,10 +601,10 @@ def test_return_unmatched_transactions_extracts_relevant_transactions_and_instru
                 "base_currency": "base_currency",
             },
             file_type="portfolio",
-            mapping_optional={"created": "txn_trade_date"},
+            mapping_optional={"created": "$2000-01-01"},
```

Comment:
we should probably have a test with and without a date filter with corresponding asserts

Reply:
Added an additional assertion!